### DISCUSSION TITLE
saved settings for each audio output

### DISCUSSION THREAD

**Discussion by mikumin27** [ppy/osu#17632] [Ideas] [answered]
Basically when I switch from my headphones to the speakers, that the speakers have a different offset saved then the headphones have.

**Comment by peppy** [comment, marked as answer]
We may introduce the ability to save settings profiles in the future, but it will still require that you manually switch profiles. We can't support per-device auto-switching, it's just too much.

**Reply by mikumin27** [reply]
would love to see it
